web/atlas.json

```diff
@@ -7156,7 +7155,7 @@
 {"id": "tzxbmo", "submitted_by": "ThaneBoi", "name": "Leonardo", "description": "Blue masked turtle from teenage mutant ninja turtles.", "website": "", "subreddit": "", "center": [1106.5, 223.5], "path": [[1096.5, 230.5], [1107.5, 231.5], [1114.5, 225.5], [1117.5, 217.5], [1106.5, 214.5], [1095.5, 225.5], [1095.5, 225.5]]},
 {"id": "tzxagt", "submitted_by": "ThaneBoi", "name": "Michelangelo", "description": "Orange masked turtle from teenage mutant ninja turtles.", "website": "", "subreddit": "", "center": [1158.5, 236.5], "path": [[1148.5, 240.5], [1162.5, 243.5], [1167.5, 242.5], [1169.5, 236.5], [1164.5, 231.5], [1153.5, 230.5], [1146.5, 234.5]]},
 {"id": "tzx99p", "submitted_by": "ThaneBoi", "name": "Donatello", "description": "Purple masked turtle from teenage mutant ninja turtles.", "website": "", "subreddit": "", "center": [1124.5, 236.5], "path": [[1114.5, 242.5], [1130.5, 243.5], [1137.5, 235.5], [1136.5, 232.5], [1126.5, 229.5], [1116.5, 232.5], [1110.5, 236.5], [1112.5, 241.5]]},
-{"id": "tzwws7", "submitted_by": "naliliV2", "name": "OptiFine", "description": "OptiFine is a Minecraft optimization mod. It allows Minecraft to run faster and look better with full support for high-definition textures and many configuration options.", "website": "https://optifine.net/home", "subreddit": "/r/Optifine", "center": [810.5, 669.5], "path": [[804.5, 664.5], [807.5, 664.5], [807.5, 661.5], [815.5, 661.5], [815.5, 677.5], [804.5, 677.5]]},
+{"id": "tzwws7", "submitted_by": "gryph667", "name": "Grandmaster Gryph", "description": "A depiction of a harpy eagle, the preferred profile pic of the Grandmaster of the April Knights during Place 2022.", "website": "https://aprilknights.org", "subreddit": "/r/Gryph667", "center": [810.5, 669.5], "path": [[804.5, 664.5], [807.5, 664.5], [807.5, 661.5], [815.5, 661.5], [815.5, 677.5], [804.5, 677.5]]},

```

Comment:
The previous entry is valid as it was present on an earlier canvas version. Post the new entry on Reddit to add it in.